examples/kubeflow/tfjob/README.md

```diff
@@ -136,14 +151,16 @@ Kubeflow installed with Pachyderm using the defaults for a joint deployment:
 Kubeflow in the `kubeflow` namespace and Pachyderm in the `pachyderm` namespace.
 
 1. Deploy Pachyderm and Kubeflow.
-2. Create a repo in Pachyderm using the command  `pachctl create repo testrepo`.
-3. Create a branch in that repo using the command `pachctl create branch testrepo@master`.
-4. Deploy the manifest to your Kubeflow using  `kubectl apply -f tf_job_s3_gateway.yaml`
-5. Using your Kubeflow TFJob dashboard or `kubectl`, monitor the TFJob pod created until it completes.
+2. Create a repo in Pachyderm using the command  `pachctl create repo inputrepo`.
+3. Add the mnist data to that repo using the command `pachctl put file inputrepo@master:/data -r -f mnist/`
```

Comment:
User should be told to either close this repo, grab the data from <URL where mnist data set is hosted by kubeflow>, or maybe we could put the data in a directory here and use the GitHub raw link with `put file -r`.

I think the 3rd option is the easiest if it works.

Reply:
I've clarified these instructions. I can't add the raw url until after the entire thing is committed. so going to merge this and then take a look.